bootloader/bootloader.go

```diff
@@ -95,13 +95,42 @@ type RecoveryAwareBootloader interface {
 	SetRecoverySystemEnv(recoverySystemDir string, values map[string]string) error
 }
 
+// ExtractedRunKernelImageBootloader is a Bootloader that also supports specific
+// methods needed to setup booting from an extracted kernel, which is needed to
+// implement encryption and/or secure boot. Prototypical implementation is UC20
+// grub implementation with FDE.
 type ExtractedRunKernelImageBootloader interface {
 	Bootloader
-	EnableKernel(snap.PlaceInfo) error    // makes the symlink
-	EnableTryKernel(snap.PlaceInfo) error // makes the symlink
-	Kernel() (snap.PlaceInfo, error)      // gives the symlink
-	TryKernel() (snap.PlaceInfo, error)   // gives the symlink (if exists)
-	DisableTryKernel() error              // removes the symlink
+
+	// EnableKernel enables the specified kernel on ubuntu-boot to be used
+	// during normal boots. The specified snap should already have been
+	// extracted.
```

Comment:
this should mention symlinking as possible implementation like EnableTryKernel does.

Reply:
Fixed now. Rephrased this and EnableTryKernel, felt like wording was a  little off.